app/controllers/contact_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class ContactController < ApplicationController
+  def question
+    @form = AskAQuestionForm.new(question_params)
+    if @form.valid? && @form.submit
+      flash[:success] = 'Your question has been submitted'
+
+      return render "question_success" if request.xhr?
```

Comment:
Are the xhr and js responses necessary?

Reply:
As far as I can tell, no (this was stolen code), so I've taken them out